src/sass/page-template-styles.scss

```diff
@@ -397,6 +433,7 @@
   .card,
   .check,
   .paypal {
+    flex-basis: 30%;
```

Comment:
Try calc(100% / 3) for perfectly 1/3 buttons.

Reply:
Done